proposed/2022/vulnerabilities-in-restore.md

```diff
@@ -80,6 +80,14 @@ This feature will be opt-in to start and gather feedback from developers.
 
 To enable the feature, a developer can add `<NuGetAudit>enable</NuGetAudit>` to their project file as a MSBuild property. To disable the feature, a developer can add `<NuGetAudit>disable</NuGetAudit>` or remove the property from the project file.
 
+#### Setting Vulnerability Auditing Modes
+
+There will be different modes to audit vulnerabilities based on the developer's or developer's team preference. To do this, a developer will opt-in to a feature called `<NuGetAuditMode>` which will have different modes such as `direct`, and `all`.
+
+These modes should be pretty straight-forward. `direct` will scan for any top-level vulnerabilities, and `all` will scan for both top-level and transitive-level vulnerabilities. The default will be `direct` until the experience is ready to be `all` given that transitive vulnerabilities are the majority of vulnerability notices (90%+).

```

Comment:
Not sure if it needs to be called out in this spec, but we should make sure we can capture a few metrics regardless of the audit  level.

- Capture number of direct vuln + potentially hashes.
- Capture number of transitive + potentially hashes. This should happen even if direct is the audit level, at least in the initial version. The perf hit is going to be negligible, but the information would be invaluable.

Reply:
I see that a too much information about implementation details for this more customer focused feature spec.

Reply:
Fair, resolving.